nemo/core/classes/modelPT.py

```diff
@@ -476,7 +476,7 @@ def restore_from(
             raise FileNotFoundError(f"Can't find {restore_path}")
 
         global _MODEL_RESTORE_PATH
-        _MODEL_RESTORE_PATH = restore_path
+        _MODEL_RESTORE_PATH = os.path.abspath(restore_path)
```

Comment:
how about `os.path.expanduser` so we can properly handle '~'?

Reply:
Oh yeah, good catch

Reply:
Done